From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
Date: Fri, 10 Jan 2025 13:15:40 -0300
Subject: GENIO: configs: 350/510/700/1200 EVK: Enable bind command

After commit 9bc199d7ea5 ("GENIO: board: 350/510/700/1200 EVK: Drop
deprecated usb_ether_init()") USB Ethernet Gadget needs to be manually
binded using the `bind` command. Let's enable that on all Genio EVK
platforms.

Signed-off-by: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
---
 configs/genio_1200_evk_defconfig | 1 +
 configs/genio_350_evk_defconfig  | 1 +
 configs/genio_510_evk_defconfig  | 1 +
 configs/genio_700_evk_defconfig  | 1 +
 4 files changed, 4 insertions(+)

diff --git a/configs/genio_1200_evk_defconfig b/configs/genio_1200_evk_defconfig
index 111111111111..222222222222 100644
--- a/configs/genio_1200_evk_defconfig
+++ b/configs/genio_1200_evk_defconfig
@@ -36,6 +36,7 @@ CONFIG_CMD_NVEDIT_EFI=y
 CONFIG_CMD_NVEDIT_INFO=y
 # CONFIG_CMD_CRC32 is not set
 CONFIG_CMD_ADC=y
+CONFIG_CMD_BIND=y
 CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_DM=y
diff --git a/configs/genio_350_evk_defconfig b/configs/genio_350_evk_defconfig
index 111111111111..222222222222 100644
--- a/configs/genio_350_evk_defconfig
+++ b/configs/genio_350_evk_defconfig
@@ -35,6 +35,7 @@ CONFIG_CMD_BOOTEFI_SELFTEST=y
 CONFIG_CMD_NVEDIT_EFI=y
 CONFIG_CMD_NVEDIT_INFO=y
 # CONFIG_CMD_CRC32 is not set
+CONFIG_CMD_BIND=y
 CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_DM=y
diff --git a/configs/genio_510_evk_defconfig b/configs/genio_510_evk_defconfig
index 111111111111..222222222222 100644
--- a/configs/genio_510_evk_defconfig
+++ b/configs/genio_510_evk_defconfig
@@ -35,6 +35,7 @@ CONFIG_CMD_BOOTEFI_SELFTEST=y
 CONFIG_CMD_NVEDIT_EFI=y
 CONFIG_CMD_NVEDIT_INFO=y
 # CONFIG_CMD_CRC32 is not set
+CONFIG_CMD_BIND=y
 CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_DM=y
diff --git a/configs/genio_700_evk_defconfig b/configs/genio_700_evk_defconfig
index 111111111111..222222222222 100644
--- a/configs/genio_700_evk_defconfig
+++ b/configs/genio_700_evk_defconfig
@@ -37,6 +37,7 @@ CONFIG_CMD_NVEDIT_EFI=y
 CONFIG_CMD_NVEDIT_INFO=y
 # CONFIG_CMD_CRC32 is not set
 CONFIG_CMD_ADC=y
+CONFIG_CMD_BIND=y
 CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_DM=y
-- 
Armbian

